Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM php:5.6
+
+RUN echo "date.timezone = UTC" > /usr/local/etc/php/conf.d/date.ini
+
+VOLUME /app
+WORKDIR /app
+
+ENTRYPOINT ["./app/console"]
+CMD ["server:run", "0.0.0.0:8000"]
```

Comment:
Yes, it makes sense. However, I think almost every developer should change `80` to something different because he most probably also has an installed server which already listening `80` on host machine.